recipes/pyterate/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "pyterate" %}
+{% set version = "2.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/FabriceSalvaire/Pyterate/archive/v{{ version }}.tar.gz
+  # run openssl sha256 <file name>
+  sha256: d0c62ae3697e47bf2f34a2d7b3dd8e3f503d88d0538098988b4813e7afcebfbd
+  # git_url: https://github.com/FabriceSalvaire/Pyterate
+  # git_rev: a394beaf44ffed0df5a845b7817b16965915cf84 # "v{{ version }}"
+  # git_depth: 1 # (Defaults to -1/not shallow)
+
+build:
+  # noarch: python
```

Comment:
I think this can probably be `noarch: python`. If so, remove the `skip` line below and then change `python` to `python >=3.5` under `host` and `run` in the `requirements` section.